server/filter.go

```diff
@@ -159,7 +159,7 @@ func (f *snapshotCountFilter) FilterTarget(store *storeInfo) bool {
 // storageThresholdFilter ensures that we will not use an almost full store as a target.
 type storageThresholdFilter struct{}
 
-const storageRatioThreshold = 0.8
+const storageRatioThreshold = 0.2
```

Comment:
storageRatio looks like current storage size / storage capacity, but here is for available storage size / storage capacity, maybe we should rename to storageAvailableRatio?